hvac/v1/__init__.py

```diff
@@ -592,7 +592,7 @@ def auth_tls(self, mount_point='cert', use_token=True):
         :return:
         :rtype:
         """
-        return self.login('/v1/auth/{0}/login'.format(mount_point), use_token=use_token)
+        return self.auth.cert.login(mount_point=mount_point, use_token=use_token)
```

Comment:
Since we're updating this extant call site to use the new class, could you please tack on a related decorator for this `auth_tls()` method? E.g.:

```python
    @utils.deprecated_method(
        to_be_removed_in_version='0.13.0',
        new_method=api.auth_methods.Cert.login,
    )
```

(This `Client` class is a bit unwieldy, so we've been taking any opportunity to move folks off of using its methods where possible. Related reference: https://github.com/hvac/hvac/blob/develop/CONTRIBUTING.md#backwards-compatibility-breaking-changes )